### PR TITLE
[extractor/crunchyroll] Add language filter for playlist

### PR DESCRIPTION
**Pull request by Chrissi2812**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

This will reintroduce the `language` filter, that has been available for the old Crunchyroll layout.
Usage is the same `--extractor-args crunchyroll:language=de-DE` as of the old extractor.

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

**Comment by pukkandan** [comment]
Instead, simply return it as `language` in the infodict. Then --match-filter can be used

**Comment by Chrissi2812** [comment]
Okay I changed it to add the `audio_locale` as `language` to the infodict and updated the tests.
Should this field also be added to single episode extractor?

**Comment by pukkandan** [comment]
Don't they already have `language` for each audio format?

**Comment by Chrissi2812** [comment]
Ahh yes, it's already in the `formats` field. 

Is the PR okay our should I squash it or anything?

File: yt_dlp/extractor/crunchyroll.py [hunk 1]
```diff
@@ -250,6 +250,11 @@ class CrunchyrollBetaShowIE(CrunchyrollBaseIE):
         'info_dict': {
             'id': 'GY19NQ2QR',
             'title': 'Girl Friend BETA',
+            'entries': [
+                {
+                    'language': 'ja-JP',
+                },
+            ] * 10
```
Comment (pukkandan) [inline yt_dlp/extractor/crunchyroll.py:257]:
```suggestion
```

**Pull request merged**